deployment/terraform/batch.tf

```diff
@@ -0,0 +1,82 @@
+#
+# Batch resources
+#
+# Pull the image ID for the latest Amazon ECS GPU-optimized AMI
+# https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-optimized_AMI.html#w520aac22c15c31b5
+data "aws_ssm_parameter" batch_gpu_container_instance_image_id {
+  name = "/aws/service/ecs/optimized-ami/amazon-linux-2/gpu/recommended/image_id"
+}
+
+resource "aws_launch_template" "batch_gpu_container_instance" {
+  block_device_mappings {
+    device_name = "/dev/xvda"
+
+    ebs {
+      volume_size = var.batch_gpu_container_instance_volume_size
+      volume_type = "gp2"
+    }
+  }
+
+  image_id = data.aws_ssm_parameter.batch_gpu_container_instance_image_id.value
```

Comment:
Given our discussion with regard to the details of how Batch CEs and launch templates play together, should we move this configuration setting out of the launch template and into the Batch CE directly?